specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/application/fetch/filter.change.none/client.rpt

```diff
@@ -0,0 +1,98 @@
+#
```

Comment:
I don't like the name of this scenario, but I couldn't come up with the perfect one. Any suggestion?

Reply:
This scenario name is okay.